job/server/src/main/java/alluxio/master/job/tracker/CmdJobTracker.java

```diff
@@ -270,4 +289,29 @@ public CmdStatusBlock getCmdStatusBlock(long jobControlId)
             .collect(Collectors.toList());
     return new CmdStatusBlock(cmdInfo.getJobControlId(), blockList, cmdInfo.getOperationType());
   }
+
+  private void cleanUnableTracedJobs() {
+    long currentTime = System.currentTimeMillis();
+    for (Map.Entry<Long, CmdInfo> x : mInfoMap.entrySet()) {
```

Comment:
Will the mCoordinators in the PlanTracker class also occupy a lot of memory? If the capacity is set too small, will it cause abnormal results when getting status?

  https://github.com/Alluxio/alluxio/blob/8dc6653ddb86b92be6c4ea1cffe0d963e1c37d7c/job/server/src/main/java/alluxio/master/job/JobMaster.java#L580